interop/BrowserDockerfile

```diff
@@ -4,10 +4,11 @@ FROM mcr.microsoft.com/playwright
 
 COPY --from=node-js-libp2p-head /app/ /app/
 WORKDIR /app/interop
-RUN npx playwright install
```

Comment:
Why is it we no longer need to install the playwright deps?

Reply:
playwright-test does it automatically if needed